app/assets/javascripts/jquery.dotdotdot-1.5.1.js

```diff
@@ -1,569 +0,0 @@
-/*	

```

Comment:
Wow, the vendor assets issue didn't sink in until I saw this... this is *terrible*!